docs/Developers/Newcomers_Guide.rst

```diff
@@ -203,6 +203,9 @@ to prevent them.
 Step 5. Sending Your Changes
 ----------------------------
 
+Before starting to write your first commit, check out this link
+http://coala.io/commit.
```

Comment:
`check this link https://...` looks worse then `check out [our documentation about. ...]` IMO. Also I don't see people clicking that link. You didn't provide a reason, nothing to look for. Even I'm confused by this.